aws/resource_aws_autoscaling_group_test.go

```diff
@@ -4105,3 +4140,57 @@ resource "aws_autoscaling_group" "test" {
 }
 `, rName)
 }
+
+func testAccAWSAutoScalingGroupPartitionConfig(rName string) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {
+  # t2.micro is not supported in us-west-2d
+  blacklisted_zone_ids = ["usw2-az4"]
+  state                = "available"
+
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
+}
+
+data "aws_ami" "test_ami" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn-ami-hvm-*-x86_64-gp2"]
+  }
+}
+
+resource "aws_launch_template" "this" {
+  name          = %[1]q
+  image_id      = data.aws_ami.test_ami.id
+  instance_type = "m5.large"
+
+  placement {
+    tenancy    = "default"
+    group_name = aws_placement_group.test.id
+  }
+}
+
+resource "aws_placement_group" "test" {
+  name     = %[1]q
+  strategy = "cluster"
+}
+
+resource "aws_autoscaling_group" "test" {
+  name_prefix        = "test"
+  availability_zones = ["${data.aws_availability_zones.available.names[0]}"]
+  desired_capacity   = 1
+  min_size           = 1
+  max_size           = 3
```

Comment:
We don't need to actually spin up instances for the Launch Template errors to occur. 👍 Verified locally.

```suggestion
  min_size           = 0
  max_size           = 0
```